ownCloudAppShared/Client/User Interface/SortBar.swift

```diff
@@ -35,13 +35,45 @@ public class SegmentedControl: UISegmentedControl {
 	}
 }
 
+public enum SearchScope : Int, CaseIterable {
+	case global
+	case local
+
+	var label : String {
+		var name : String!
+
+		switch self {
+			case .global: name = "all".localized
+			case .local: name = "folder".localized
```

Comment:
Strings missing in `Localizable.strings`

Reply:
Fixed with the latest commits.